routing/routing_benchmarks/bicycle_routing_tests.cpp

```diff
@@ -44,4 +44,11 @@ UNIT_CLASS_TEST(BicycleTest, Smoke)
   m2::PointD const final = MercatorBounds::FromLatLon(55.79369, 37.56054);
   TestRouters(start, final);
 }
+
+UNIT_CLASS_TEST(BicycleTest, MailRuOfficeToUlSimanova5K2)
```

Comment:
I agree MailRuOfficeToUlSimanova5K2 is a bad address specification. I suggest using name specification which I use in routing_integration_tests: https://github.com/mapsme/omim/pull/4307